packages/language/src/index.js

```diff
@@ -434,3 +434,4 @@ Language.prototype.text = function(content, options) {
 };
 
 module.exports = Language;
+module.exports.v1beta1 = require('./v1beta1');
```

Comment:
(RE: #1492)

We should probably put some documentation here to show the user what they're getting. Should we wait to see how #1492 resolves, or put up something basic for now to make this merge-able?